bin/morgue.js

```diff
@@ -3632,6 +3632,194 @@ function samplingUsage(str) {
+    config.reset_interval = parseTimeInt(argv["reset-interval"]);
+  }
+
+  config.missing_symbols = {};
+  const keepWhitelisted = argv["process-whitelisted"];
+  if (keepWhitelisted !== undefined) {
+    config.missing_symbols.process_whitelisted = parseBool(keepWhitelisted);
+  }
+  const keepPrivate = argv["process-private"];
+  if (keepPrivate !== undefined) {
+    config.missing_symbols.process_private = parseBool(keepPrivate);
+  }
+
+  let bucketsUnparsed = argv.buckets;
+  if (bucketsUnparsed !== undefined) {
+    if (typeof bucketsUnparsed !== 'number') {
+      errx("--buckets must be integer");
+    }
+    config.buckets = bucketsUnparsed;
+  }
+
+  const resetIntervalUnparsed = argv["reset-interval"];
+  if (resetIntervalUnparsed !== undefined) {
+    config.reset_interval = parseTimeInt(resetIntervalUnparsed);
+  }
+  return config;
+}
+
+function samplingConfigure(coroner, argv, config) {
+  let universe = argv.universe;
+  if (!universe) {
+    universe = Object.keys(config.config.universes)[0];
+  }
+
+  let project = argv.project;
+
+  if (!universe) {
+    errx("--universe is required");
+  }
+
+  if (!project) {
+    errx("--project is required");
+  }
+
+  let bpg = coronerBpgSetup(coroner, argv);
+  let model = bpg.get();
+
+  /* Find universe. */
+  let uid = 0;
+  for (let u of model.universe) {
+    if (u.get("name") === universe) {
+      uid = u.get("id");
+    }
+  }
+
+  if (uid == 0) {
+    errx("Universe not found");
+  }
+
+  let pid = 0;
+  for (let p of model.project) {
+    if (p.get("universe") === uid && p.get("name") == project) {
+      pid = p.get("pid");
+    }
+  }
+
+  if (pid === 0) {
+    errx("Project not found");
+  }
+  
+  let disabled = 0;
+  if (argv.disable) {
+    disabled = 1;
+  }
+
+  let projectSampling = null;
+  if (model.project_sampling) {
+    for (let cfg of model.project_sampling) {
+      if (cfg.get("project") === pid) {
+        projectSampling = cfg;
+      }
+    }
+  }
+
+  if (argv.clear) {
+    if (projectSampling) {
+      bpg.delete(projectSampling);
+      bpg.commit();
+    }
+    console.log(`Sampling configuration cleared. Project ${ project } will use coronerd.conf defaults.`);
+    return;
+  }
+
+  let configurationObj = {};
+  if (projectSampling) {
+    configurationObj = JSON.parse(projectSampling.get("configuration"));
+  }
+
+  /*
+   * Allow --disable by itself, by not trying to parse the config
+   */
+  if (disabled === 0) {
+    configurationObj = samplingConfigFromArgv(argv);
+  }
+
+  const configuration = JSON.stringify(configurationObj);
+  if (projectSampling) {
+    bpg.modify(projectSampling, { disabled, configuration });
+  } else {
+    const obj = bpg.new("project_sampling").withFields({
+      project: pid,
+      configuration,
+      disabled
+    });
+    bpg.create(obj);
+  }
+
+  bpg.commit();
+  console.log("Sampling configuration applied");
+  if (disabled == 1) {
+    console.log(`Project ${ project } now has sampling explicitly disabled.
```

Comment:
are we setting the whole paragraph to yellow? That's a lot and hard to read. I would highlight only the call to action ("To use coronerd.conf defaults, use --clear instead") at the end.

Also, we should end that sentence with a period.